content/docs/dvclive/how-it-works.md

```diff
@@ -76,3 +81,10 @@ your repo.
 If you don't have a DVC pipeline, you can include
 [`save_dvc_exp=True`](/doc/dvclive/api-reference/live#parameters) to save the
 results as a DVC experiment.
+
+When using `Live.log_artifact("model.pt")`, DVCLive will
+[cache](/doc/start/data-management/data-versioning) the `model.pt` file with DVC
+to avoid tracking large artifacts in Git. It will generate a `model.pt.dvc`
+metadata file, which you should track in Git. You can
+[retrieve](/doc/start/data-management/data-versioning#retrieving) the artifact
+from the Git commit.
```

Comment:
Thanks for including my suggestion, but I think it feels a little disconnected. Feel free to drop it and we can do a follow-up PR if that's easier.

I think we should rework this whole section. Proposed text:

```
# Track the results

DVCLive expects each run to be tracked by Git, so it will save each run to the
same path and overwrite the results each time. Include
[`save_dvc_exp=True`](/doc/dvclive/api-reference/live#parameters) to 
auto-track in Git as a <abbr>DVC experiment</abbr>. DVC experiments
are Git commits that DVC can find but that don't clutter your Git history 
or create extra branches in your repo.

### Track artifacts

Models and data are often large and aren't easily tracked in Git. 
`Live.log_artifact("model.pt")` will [cache](/doc/start/data-management/data-versioning)
the `model.pt` file with DVC and make Git ignore it. It will generate a `model.pt.dvc`
metadata file, which can be tracked in Git and becomes part of the experiment.
You can [retrieve](/doc/start/data-management/data-versioning#retrieving) the
versioned artifact from the Git commit.

### Run experiments with `dvc exp run`

You may [run experiments](/doc/user-guide/experiment-management/running-experiments)
using DVC <abbr>pipelines</abbr>. Once you setup your pipeline, you can run it with `dvc exp run`.
This will track the inputs and outputs of your code, cache them so you never waste time repeating 
steps, and enable other features like queuing, hyperparameter tuning, and
grid searches.

Reply:
Related: https://github.com/iterative/dvclive/issues/468#issuecomment-1444219085 cc @shcheklein